### ISSUE TITLE
Feature request: Human-readable direct-to-printer links

### ISSUE THREAD

**Issue by WayneManion** [ballaswag/guppyflo#4]
One of the reasons I installed guppyflo was to be able to jump directly to a printer via a weblink. I run multiple printers on the same Klipper host. Without guppyflo I have to open the fluidd interface on the host machine, then use the menus to specify the printer I want to work with. 

With guppyflo I can use a link to go directly to the fluidd interface for one printer. An example of such a link is:
http://klipperhost.local:9873/printers/3761410498/fluidd/#/

This is most of the way to the "ideal" solution. I would like to be able to use a link like "http://klipperhost.local/printerName/fluidd" and jump directly to fluidd for my printer of choice. 

Thanks.

**Comment by ballaswag** [comment]
Yeah these URLs are long. I need to figure the wildcard stuff for path matching to shorten them. For now the pattern is needed to get the go mux working.
